### PR TITLE
Extracted gradle dependencies to separate file.

### PR DESCRIPTION
**Pull request by andriy1507**
(cherry picked from commit 4fb000c63b233175c5114113463c4d79f8f0a0fa)

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -0,0 +1,107 @@
+//Android
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+def appCompatVersion = '1.2.0-alpha01'
+def coreVersion = '1.2.0-rc01'
+
+//Gson
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+def gsonVersion = '2.8.6'
+
+//UI
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+def recyclerviewVersion = '1.2.0-alpha01'
+def constraintlayoutVersion = '2.0.0-beta4'
+def legacySupportVersion = '1.0.0'
+def lifecycleVersion = '2.2.0-rc03'
+
+//Coroutines
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+def coroutinesVersion = '1.3.3'
+
+//Retrofit
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+def retrofitVersion = '2.7.0'
+
+//OkHttp
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+def okhttpVersion = '4.2.2'
+
+//ZonedDateTime
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+def threetenbpVersion = '1.4.0'
+
+//Tests
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+//Gradle
+
+ext.gradleVersion = '3.5.3'
+
+//Kotlin
+
+ext.kotlinVersion = '1.3.61'
+
+//SDK
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+def junitVersion = '4.13-rc-2'
+def junitAndroidVersion = '1.1.2-alpha03'
+def espressoVersion = '3.3.0-alpha03'
+
+//Gradle
+
+ext.gradleVersion = '3.5.3'
+
+//Kotlin
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+def threetenbpVersion = '1.4.0'
+
+//Tests
+
+def junitVersion = '4.13-rc-2'
+def junitAndroidVersion = '1.1.2-alpha03'
+def espressoVersion = '3.3.0-alpha03'
+
+//Gradle
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: dependencies.gradle [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+ext.javaVersion = JavaVersion.VERSION_1_8
+
+//Configurations
+ext.versionCode = 1
+ext.versionName = '1.0'
+ext.appId = "com.goryachok.forecastapp"
+ext.testRunner = "androidx.test.runner.AndroidJUnitRunner"
+
+//Dependencies
+
```
Comment (EugeneKryvoshey) [inline dependencies.gradle]: Please, remove empty line.

File: build.gradle [hunk 13]
```diff
@@ -1,15 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    // Dependencies
+    apply from: "./dependencies.gradle"
+
     repositories {
         google()
         jcenter()
```
Comment (EugeneKryvoshey) [inline build.gradle]: Please, remove empty line.

File: app/build.gradle [hunk 14]
```diff
@@ -22,52 +22,27 @@ android {
...
+    implementation appDependencies.android
+    implementation appDependencies.ui
+    implementation appDependencies.api
+    implementation appDependencies.coroutines
+    implementation appDependencies.utils
 
-    //ZonedDateTime
-    implementation 'org.threeten:threetenbp:1.4.0'
+    testImplementation appDependencies.test
+    androidTestImplementation appDependencies.androidTest
```
Comment (EugeneKryvoshey) [inline app/build.gradle]: Please, remove empty line.

**Review by EugeneKryvoshey — CHANGES_REQUESTED** [review]

File: dependencies.gradle (hunk 1 above)
Comment (andriy1507) [reply dependencies.gradle]: +

File: dependencies.gradle (hunk 2 above)
Comment (andriy1507) [reply dependencies.gradle]: +

File: dependencies.gradle (hunk 3 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: dependencies.gradle (hunk 4 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: dependencies.gradle (hunk 5 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: dependencies.gradle (hunk 10 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: dependencies.gradle (hunk 11 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: dependencies.gradle (hunk 12 above)
Comment (andriy1507) [reply dependencies.gradle]: ok

File: build.gradle (hunk 13 above)
Comment (andriy1507) [reply build.gradle]: ok

File: app/build.gradle (hunk 14 above)
Comment (andriy1507) [reply app/build.gradle]: ok

**Comment by andriy1507** [comment]
Removed empty lines.

**Review by EugeneKryvoshey — APPROVED** [review]

**Pull request merged**